src/components/Titles.js

```diff
@@ -1,22 +1,40 @@
-// import React from "react";
-// import { Link } from "react-router-dom";
+import React from "react";
+import { NavLink } from "react-router-dom";
 
-// function Titles() {
-//   return (
-//     <div className="content__container">
-//       <div className="container__titles">
-//           <h2>
-//             <span>NETFLIX CLONE</span>
-//           </h2>
-//           <h2>
-//             <span>CRYPTO TRACKER</span>
-//           </h2>
-//           <h2>
-//             <span>IMAGE MAKER</span>
-//           </h2>
-//       </div>
-//     </div>
-//   );
-// }
+function Titles({ clickHandler, active }) {
+  return (
+    <div className="container__titles">
+      <>
+        <h2>
+          <NavLink
+            to="/Netflix"
+            exact
+            style={{ textDecoration: "none", color: "inherit" }}
+          >
+            <span>NETFLIX CLONE</span>
+          </NavLink>
+        </h2>
```

Comment:
Should make this a component and pass in the props

Reply:
Created array of objects (h2__titles) and rendered h2/NavLink elements utilizing .map() function.

Reply:
Used the same method in App.js for routing, also surrounded each component with divs of className page in order to utilize onEnter and onExit events from react-transition-group.